docs/guides/self-hosted/development.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Development
+description: Learn how to set up a local Latitude instance for development
+---
+
+## Prerequisites:
+
+- Install [PNPM](https://pnpm.io/installation).
+- Install [Tmuxinator](https://github.com/tmuxinator/tmuxinator) (`brew install tmuxinator` on Mac).
+- Install [Docker](https://docs.docker.com/desktop/install/mac-install) and docker-compose (or a compatible alternative).
```

Comment:
We use https://orbstack.dev/ and is way superior. Can you link here as an alternative please?

Reply:
We should promote tech that works great. We all use it because is way better than standard Docker client

Reply:
let's not debate this please

Reply:
I'll leave it to txus's judgment

Reply:
Since I mentioned the compatible alternative, I think that's ok, it's left for advanced users (I use podman & podman compose). In the end, tmuxinator calls `docker compose`, so it needs to be anything compatible with that interface, doesn't matter too much what it is.